automated_packaging/prepare_changelog.pl

```diff
@@ -0,0 +1,60 @@
+#!usr/bin/perl
+$CODE_PATH=$ARGV[0];
+$VERSION=$ARGV[1];
+$PROJECT=$ARGV[2];
+$SHA_CODE_OF_LAST_COMMIT=$ARGV[3];
+
+# Name of the repo is represented differently on logs and repos
+$repo_log_name="citus";
+$master_branch_name="master";
+
+# Necessary to write log date
+my @abbr_mon = qw(January February March April May June July August September October November December);
+my ($sec,$min,$hour,$mday,$mon,$year,$wday,$yday,$isdst) = localtime(time);
+$year += 1900;
+
+# Move to the packaging folder
+chdir("$CODE_PATH") or die "$!";
+$release_major_minor = substr($VERSION, 0, 3);
+
+# Checkout the master branch
+system("git checkout $master_branch_name")
+
+# Read the current changelog
+open(CHANGELOG_FILE, "<CHANGELOG.md") || die "File not found";
+my @changelog_current_lines = <CHANGELOG_FILE>;
+close(CHANGELOG_FILE);
+
+# Update the changelog file of the debian branch
+open(CHANGELOG_FILE, ">CHANGELOG.md") || die "File not found";
+print CHANGELOG_FILE "### $repo_log_name v$VERSION ($abbr_mon[$mon] $mday, $year) ###\n\n";
+
+@log_output = `git log`;
+foreach $line (@log_output) {
+	$line =~ s/^\s+//;
+
+	# Break the loop if the commit's sha code is equal to the given sha code
```

Comment:
What if last commit has a DESCRIPTION info? I think this section should go after writing DESCRIPTION to CHANGELOG section.